app/views/users/new.html.erb

```diff
@@ -1,2 +1,5 @@
-<h1>Users#new</h1>
-<p>Find me in app/views/users/new.html.erb</p>
+<%= form_for @user, url: {controller: "users", action: "create"} do |f| %>
+  <%= f.label "Hi! What's your name?" %>
+  <%= f.text_field :name %>
+  <%= f.submit "Play!" %>
```

Comment:
yup! so I have root going to this signin form that will redirect to the button page. Eventually this should be a modal or something